src/Impl/CopyGitLink.Shared/Core/GitOnlineServices/GitHubnLab.cs

```diff
@@ -70,29 +74,36 @@ var relativePath
 
             var repositoryName = repositoryInfo.Properties[Repository];
             var organization = repositoryInfo.Properties[Organization];
+            var host = repositoryInfo.Properties[Host];
 
             Requires.NotNullOrEmpty(organization, nameof(organization));
             Requires.NotNullOrEmpty(repositoryName, nameof(repositoryName));
+            Requires.NotNullOrEmpty(host, nameof(host));
             Requires.NotNullOrEmpty(branchName, nameof(branchName));
 
             // Link to a file without line to select.
-            string url = $"https://github.com/{organization}/{repositoryName}/blob/{branchName}/{relativePath}";
+            string url = $"https://{host}/{organization}/{repositoryName}/blob/{branchName}/{relativePath}";
 
-            if (startLineNumber.HasValue && endLineNumber.HasValue)
+            if (endLineNumber.HasValue)
```

Comment:
nevermind, it doesn't matter. However, I realized that if we do a selection from the bottom to the top, the endLineNumber and startLineNumber are inverted. That's my bad. if you want to, it would be nice to fix it. :)

Reply:
yeap indeed, i'll have a look.